### ISSUE TITLE
Bibliocraft 2.4.4 crashes

### ISSUE THREAD

**Issue by KillerDan3000** [Nuchaz/BiblioCraft#574]
Hi, I've been enjoying my use of bibliocraft on my newly modded server. Unfortunately, I've recently started to disconnect immediately after login and I have traced it back to Bibliocraft. I can log in and stay connected without the mod in my server's mods folder, but I'd really love it if we could find a solution which allows me to play and use the mod too. Please see a copy of my latest crash log below and let me know if you have any insight into the issue.

-Dan

[latest.log](https://github.com/Nuchaz/BiblioCraft/files/2060862/latest.log)

**Comment by KillerDan3000** [comment]
That worked 100%. I can totally live without the clipboard, considering all of the other amazing items that I get when I combine Bibliocraft, Chisel, and Storage Drawers. Thanks a lot!

**Comment by KillerDan3000** [comment]
That worked 100%. I can totally live without the clipboard, considering all of the other amazing items that I get when I combine Bibliocraft, Chisel, and Storage Drawers. Thanks a lot!

**Comment by Nuchaz** [comment]
Thanks for the crash report, there is a very good chance this is related to #570 . I have an idea for a fix and will implement for next release.

**Comment by desvandev** [comment]
2.4.4 log

```
Time: 6/10/18 3:55 PM
Description: Tesselating block model

java.lang.NullPointerException: Tesselating block model
	at net.minecraftforge.client.model.obj.OBJModel$OBJBakedModel.buildQuads(OBJModel.java:1345)
	at net.minecraftforge.client.model.obj.OBJModel$OBJBakedModel.func_188616_a(OBJModel.java:1314)
	at jds.bibliocraft.models.ModelClipboard.func_188616_a(ModelClipboard.java:336)
	at net.minecraft.client.renderer.BlockModelRenderer.func_187497_c(BlockModelRenderer.java:157)
	at net.minecraftforge.client.model.pipeline.ForgeBlockModelRenderer.func_187497_c(ForgeBlockModelRenderer.java:68)
	at net.minecraft.client.renderer.BlockModelRenderer.func_187493_a(BlockModelRenderer.java:78)
	at net.minecraft.client.renderer.BlockModelRenderer.func_178267_a(BlockModelRenderer.java:57)
	at net.minecraft.client.renderer.BlockRendererDispatcher.func_175018_a(BlockRendererDispatcher.java:80)
	at codechicken.lib.render.block.CCBlockRendererDispatcher.func_175018_a(CCBlockRendererDispatcher.java:69)
	at com.elytradev.architecture.client.render.CustomBlockDispatcher.func_175018_a(CustomBlockDispatcher.java:98)
	at net.minecraft.client.renderer.chunk.RenderChunk.func_178581_b(RenderChunk.java:327)
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.func_178474_a(SourceFile:100)
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:43)
	at java.lang.Thread.run(Unknown Source)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.minecraftforge.client.model.obj.OBJModel$OBJBakedModel.buildQuads(OBJModel.java:1345)
	at net.minecraftforge.client.model.obj.OBJModel$OBJBakedModel.func_188616_a(OBJModel.java:1314)
	at jds.bibliocraft.models.ModelClipboard.func_188616_a(ModelClipboard.java:336)
	at net.minecraft.client.renderer.BlockModelRenderer.func_187497_c(BlockModelRenderer.java:157)
	at net.minecraftforge.client.model.pipeline.ForgeBlockModelRenderer.func_187497_c(ForgeBlockModelRenderer.java:68)

-- Block model being tesselated --
Details:
	Block: bibliocraft:clipboard
	Block location: World: (19999,131,1152), Chunk: (at 15,8,0 in 1249,72; contains blocks 19984,0,1152 to 19999,255,1167), Region: (39,2; contains chunks 1248,64 to 1279,95, blocks 19968,0,1024 to 20479,255,1535)
	Using AO: false
Stacktrace:
	at net.minecraft.client.renderer.BlockModelRenderer.func_187493_a(BlockModelRenderer.java:78)
	at net.minecraft.client.renderer.BlockModelRenderer.func_178267_a(BlockModelRenderer.java:57)

-- Block being tesselated --
Details:
	Block type: ID #1883 (tile.BiblioCraft:Clipboard // jds.bibliocraft.blocks.BlockClipboard)
	Block data value: 0 / 0x0 / 0b0000
	Block location: World: (19999,131,1152), Chunk: (at 15,8,0 in 1249,72; contains blocks 19984,0,1152 to 19999,255,1167), Region: (39,2; contains chunks 1248,64 to 1279,95, blocks 19968,0,1024 to 20479,255,1535)
Stacktrace:
	at net.minecraft.client.renderer.BlockRendererDispatcher.func_175018_a(BlockRendererDispatcher.java:80)
	at codechicken.lib.render.block.CCBlockRendererDispatcher.func_175018_a(CCBlockRendererDispatcher.java:69)
	at com.elytradev.architecture.client.render.CustomBlockDispatcher.func_175018_a(CustomBlockDispatcher.java:98)
```

**Comment by Nuchaz** [comment]
I just released v2.4.5 of BiblioCraft which has fixed this issue.
